vgen/src/main/resources/io/gapi/vgen/php/discovery_fragment.snip

```diff
@@ -3,7 +3,123 @@
 @end
 
 @snippet generateClass(method, body, imports)
+  @let api = context.getApi, \
+       apiName = api.getName, \
+       serviceTitle = context.getService.getTitle
+    <?php
+    /*
+     * PRE-REQUISITES:
+     * ---------------
+     * 1. If not already done, enable the {@serviceTitle} and check quota for your project at
+     *    https://console.developers.google.com/apis/api/{@apiName}_component/quotas
+     * 2. To install the client library on Composer, check installation instructions at
+     *    https://github.com/google/google-api-php-client.
+     * 3. This sample uses Application Default Credentials for Auth. If not already done, install the gcloud CLI from
+     *    https://cloud.google.com/sdk/ and run 'gcloud beta auth application-default login'
+     */
+
+    // composer autoloading
+    require_once __DIR__ . '/vendor/autoload.php';
+
+    @join import : imports
+      use {@import};
+    @end
+
+    {@body}
+
+  @end
 @end
 
 @snippet generateBody(method)
+  @let apiName = context.getSimpleApiName, \
+       apiaryConfig = context.getApiaryConfig, \
+       serviceName = context.getServiceClassName, \
+       params = context.getFlatMethodParams(method)
+    function {@apiName}Example
```

Comment:
Does PHP allow for just free-floating code? If it does, you don't need to wrap this in a function.